grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -79,7 +79,11 @@ class StockMovementService {
             throw new ValidationException("Invalid stock movement", stockMovement.errors)
         }
 
-        return createRequisitionBasedStockMovement(stockMovement)
+        if(stockMovement.stockMovementType == StockMovementType.OUTBOUND) {
```

Comment:
Will this change impact the existing workflows (order, inbound)?

Reply:
It shouldn't be here. I have no idea why it's included, I was sure I've removed it but must have not added to commit. Will udpate soon.

Reply:
@jmiranda I have updated PR.